app/components/candidate_interface/course_choices_review_component.rb

```diff
@@ -62,13 +74,41 @@ def location_row(course_choice)
       }
     end
 
+    def study_mode_options_row(course)
+      {
+        key: 'Available options',
+        value: course.study_mode.humanize,
```

Comment:
Why is this slightly different from what `study_mode_row` does? That one does `.offered_option` as well 🤔

Reply:
You're correct it shouldn't be there. I had misunderstood what it was doing. and thought it selected the option the candidate chose rather than the courses study mode.